Source/JavaScriptCore/dfg/DFGPlan.cpp

```diff
@@ -369,10 +370,18 @@ Plan::CompilationPath Plan::compileInThreadImpl()
         RUN_PHASE(performCriticalEdgeBreaking);
         if (Options::createPreHeaders())
             RUN_PHASE(performLoopPreHeaderCreation);
+
+        if (Options::useFTLLoopUnrolling()) {
+            RUN_PHASE(performCPSRethreading);
+            RUN_PHASE(performDFGLoopUnrolling);
+            RUN_PHASE(performCFGSimplification);
+            RUN_PHASE(performCriticalEdgeBreaking);
+        }
```

Comment:
Curious, why loop unrolling pipeline is 
```
performCPSRethreading
performDFGLoopUnrolling
performCFGSimplification
performCriticalEdgeBreaking
```
Just wonder what are `performCPSRethreading`, `performCFGSimplification`, and `performCriticalEdgeBreaking` trying to achieve.